docs/source/tensor_attributes.rst

```diff
@@ -15,13 +15,15 @@ torch.dtype
 .. class:: torch.dtype
 
 A :class:`torch.dtype` is an object that represents the data type of a
-:class:`torch.Tensor`. PyTorch has nine different data types:
+:class:`torch.Tensor`. PyTorch has eleven different data types:
 
 ========================   ===========================================   ===========================
 Data type                  dtype                                         Tensor types
 ========================   ===========================================   ===========================
 32-bit floating point      ``torch.float32`` or ``torch.float``          ``torch.*.FloatTensor``
 64-bit floating point      ``torch.float64`` or ``torch.double``         ``torch.*.DoubleTensor``
+64-bit complex             ``torch.complex64`` or ``torch.cfloat``       ``torch.*.ComplexFloatTensor``
```

Comment:
I thought we weren't going to add e.g. `torch.*ComplexFloatTensor`?  IMO we should change the "Tensor types" header to "Legacy constructor"

Reply:
I made these changes before our discussion. will remove the changes and also updated the header title